plugins/speculation-rules/load.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Plugin Name: Speculation Rules
+ * Plugin URI: https://github.com/WordPress/performance/tree/trunk/modules/js-and-css/speculation-rules
```

Comment:
```suggestion
 * Plugin URI: https://github.com/WordPress/performance/tree/trunk/plugins/speculation-rules
```

Reply:
Good catch! I fixed this for `auto-sizes` as well.